lightly/transforms/models/simclr_transform.py

```diff
@@ -58,6 +58,7 @@ def __init__(
         min_scale: float = 0.15,
         random_gray_scale: float = 0.2,
         gaussian_blur: float = 0.5,
+        kernel_size: Optional[float] = None,
```

Comment:
I noticed that `input_size` should be `224` in `SimCLRTransform` and `SimCLRViewTransform`